powerauth-data-adapter/pom.xml

```diff
@@ -14,16 +14,16 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.0.4.RELEASE</version>
+        <version>2.0.8.RELEASE</version>
```

Comment:
Should we use `2.0.8` (Wildfly issue)?

Reply:
The issue only occurs with `spring-ws-security` in dependencies, so there is no problem in this particular project.